purge.cc

```diff
@@ -0,0 +1,267 @@
+// Copyright (c) 2017 Rockset.
```

Comment:
What's up with this file? An accident?

Reply:
I do not see this change

Reply:
What do you mean? There is an extra `purge.cc` file in the commit, isn't there?